cluster-service/deploy/openshift-templates/arohcp-service-template.yml

```diff
@@ -357,6 +357,8 @@ objects:
             version_constraints:
               min_version: 4.11.0
 
+# TODO: parameterize managedIdentitiesDataPlaneAudienceResource after templatize refactor and helm chart conversion
+# so that the value can be configured in higher envs.
```

Comment:
Is there some jira to track this ?

Reply:
None that I am aware about but @tony-schndr is progressing this is what I've been made aware of.

@tony-schndr @janboll do you know of any card ?

Reply:
There was a general card about helm chat but not a specific one about it. 
I've created it under the same epic: ARO-12801